aws/resource_aws_iam_role_policy_attachment_test.go

```diff
@@ -93,6 +98,21 @@ func testAccCheckAWSRolePolicyAttachmentAttributes(policies []string, out *iam.L
 	}
 }
 
+func testAccAWSIAMRolePolicyAttachmentImportStateIdFunc(s *terraform.State) (string, error) {
```

Comment:
Minor nitpick: For future maintainability, we should parameterize this and only depend on the resource in question (`aws_iam_role_policy_attachment`). e.g.

```go
func testAccAWSIAMRolePolicyAttachmentImportStateIdFunc(resourceName string) resource.ImportStateIdFunc {
	return func(s *terraform.State) (string, error) {
		rs, ok := s.RootModule().Resources[resourceName]
		if !ok {
			return "", fmt.Errorf("Not found: %s", resourceName)
		}

		return fmt.Sprintf("%s/%s", rs.Primary.Attributes["role"], rs.Primary.Attributes["policy_arn"]), nil
	}
}
```